snap/info.go

```diff
@@ -32,6 +36,8 @@ type Info struct {
 	Apps        map[string]*AppInfo
 	Plugs       map[string]*PlugInfo
 	Slots       map[string]*SlotInfo
+	// info for store hosted snaps
+	Store *StoreInfo
```

Comment:
I'm tempted to suggest merging StoreInfo here. Fields such as Size and Sha512 are really properties of the snap itself, rather than the download, and thus worth keeping for good. DownloadURL and IconURL also feel okay to have here.

The only odd case is LastUpdated.. What do we use it for?